NPD e2e test CI is failing

For example, see this test run:
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-npd-e2e-test/1203184740086059013

The real failure is:
```
GO111MODULE=on ginkgo -nodes=3 -mod vendor -timeout=10m -v -tags " journald" \
./test/e2e/metriconly/... -- \
-project= -zone=us-central1-a \
-image= -image-family=cos-73-lts -image-project=cos-cloud \
-ssh-user=prow -ssh-key=/etc/ssh-key-secret/ssh-private \
-npd-build-tar=`pwd`/node-problem-detector-v0.8.0-31-g9d584df.tar.gz \
-boskos-project-type=gce-project -job-name=ci-npd-e2e-test \
-artifacts-dir=/logs/artifacts
/bin/sh: 1: ginkgo: not found
make: *** [Makefile:128: e2e-test] Error 127
+ EXIT_VALUE=2
+ set +o xtrace
```

We [are using](https://github.com/kubernetes/test-infra/blob/d8a8d3fed70ff7fc35726304a6d50e4027c9bc89/config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml#L264) the container image `gcr.io/k8s-testimages/kubekins-e2e:v20191204-a3f3e16-master`. Apparently we need to install ginkgo in it.

--- a/test/build.sh
+++ b/test/build.sh
@@ -43,7 +43,7 @@ function print-help() {
   echo "  pr             Build node-problem-detector for presubmit jobs and push to staging. Flag -p is required."
   echo "  ci             Build node-problem-detector for CI jobs and push to staging."
   echo "  get-ci-env     Download environment variable file from staging for CI job."
-  echo "  install-lib    Install the libraries needed."
+  echo "  install-lib    Install the required libraries and tools for presubmit and CI jobs."
   echo
   echo "Examples:"
   echo "  build.sh help"
@@ -66,6 +66,9 @@ function get-version() {
 function install-lib() {
   apt-get update
   apt-get install -y libsystemd-dev
+  # Turn off go modules here, because we are not trying to install
+  # ginkgo library/module. We are trying to install the ginkgo executable.
+  GO111MODULE=off go get -v github.com/onsi/ginkgo/ginkgo
 }
 
 function write-env-file() {

